ee/codegen/src/__test__/nodes/generic-nodes/__snapshots__/generic-node.test.ts.snap

```diff
@@ -372,6 +372,29 @@ class MyCustomNode(BaseNode):
 "
 `;
 
+exports[`GenericNode > empty StartNode with label containing space > getNodeDisplayFile 1`] = `
+"from uuid import UUID
+
+from vellum_ee.workflows.display.editor import NodeDisplayData, NodeDisplayPosition
+from vellum_ee.workflows.display.nodes import BaseNodeDisplay
+from vellum_ee.workflows.display.nodes.types import PortDisplayOverrides
+
+from ...nodes.start import StartNode
+
+
+class StartNodeDisplay(BaseNodeDisplay[StartNode]):
+    label = "StartNode"
```

Comment:
DevinAI I would expect this label to either be the same as the one in the `node['label']`, or not here at all bc of our default logic. Please now iterate on the business logic